config/routes.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'user#index'
+  get 'users/:id', to: 'user#show'
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/', to: 'user#index'
+  get 'users/:id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] Kindly write the routes this way:
```ruby
     resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
as it's more professional and easier to read.